XenAdmin/TabPages/NICPage.Designer.cs

```diff
@@ -56,6 +56,7 @@ private void InitializeComponent()
             this.ColumnDeviceName = new System.Windows.Forms.DataGridViewTextBoxColumn();
             this.ColumnBusPath = new System.Windows.Forms.DataGridViewTextBoxColumn();
             this.ColumnFCoECapable = new System.Windows.Forms.DataGridViewTextBoxColumn();
+            this.SriovSupportedColumn = new System.Windows.Forms.DataGridViewTextBoxColumn();

```

Comment:
Other columns were all named like "ColumnXxx", we should align with them. Change the name to "ColumnSriovSupported".

Reply:
Also, the header is currently wrapping the text into two lines, while we normally go for single line headers.